Th Mapping abstract class was moved to collections.abc in python 3.3 and the
compat remove in 3.10.

--- a/bindings/python/babeltrace/babeltrace.i.in
+++ b/bindings/python/babeltrace/babeltrace.i.in
@@ -672,8 +672,8 @@
 _scopes = [CTFScope.EVENT_FIELDS, CTFScope.EVENT_CONTEXT, CTFScope.STREAM_EVENT_CONTEXT,
 	CTFScope.STREAM_EVENT_HEADER, CTFScope.STREAM_PACKET_CONTEXT, CTFScope.TRACE_PACKET_HEADER]
 
-import collections
-class Event(collections.Mapping):
+import collections.abc
+class Event(collections.abc.Mapping):
 	"""
 	This class represents an event from the trace.
 	It is obtained using the TraceCollection generator functions.
